packages/pages/src/upgrade/pagesUpdater.ts

```diff
@@ -98,13 +102,17 @@ export const updatePagesComponents = async (targetDirectory: string) => {
     "@yext/pages-components",
     null
   );
+  // update imports from sites-components to pages-components
+  replaceSitesComponentsImports(targetDirectory);
```

Comment:
This func should call this, but also do replacePagesSlashComponentsImports for this new package. We need to handle both.